ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4494,3 +4494,19 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+  - name: v2UserBulkUpload
+    request_path: "{{ user_service_prefix }}/v2/bulk/upload"
+    upstream_url: "{{ learning_service_url }}/v2/bulk/user/upload"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: userAdmin
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
```

Comment:
i think change this to "small_rate_limit_per_hour"

Reply:
ok